utils/ensemble_tools.py

```diff
@@ -0,0 +1,166 @@
+""" A module to create input directories and encode
+configfurations for FabNESO
+"""
+
+import os
+import shutil
+import itertools
+import re
+from pathlib import Path
+import lxml.etree as ET
+
+
+def create_dir_tree(
+    sweep_path,
+    n_dirs,
+    destructive,
+    copy_dir,
+    edit_file,
+    parameter_to_scan,
+    scan_range,
+    outdir_prefix,
+):
+    """Create a directory tree in the sweep_path"""
+
+    copy_files = os.path.isdir(copy_dir)
+    if not copy_files:
+        print("No correct copy dir found, just creating the tree for now!")
+
+    # Make the base directory
+    try:
+        make_directory(sweep_path, destructive)
+    except FileExistsError:
+        return
+
+    # Set the initial value of the scanned parameter to the lower limit
+    para_val = scan_range[0]
+
+    for i in range(n_dirs):
+        new_dir = Path(sweep_path) / "SWEEP" / outdir_prefix + i
```

Comment:
```suggestion
        new_dir = Path(sweep_path) / "SWEEP" / f"{outdir_prefix}{i}"
```
Currently raises `TypeError: unsupported operand type(s) for +: 'PosixPath' and 'int'`